softdev.bib

```diff
@@ -2757,3 +2757,43 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@misc{servo,
+    title = {The Servo Parallel Browser Engine Project},
+    howpublished = {\href{https://servo.org}{https://servo.org}},
```

Comment:
I notice our other entries are (some? mostly?) `\url` which avoids repeating the URL twice. I guess that's a standard part of pdflatex? [I'm too lazy to remind myself...]

Reply:
Ah yes possibly, I can change this if you like. I shamelessly copied this from the octane entry.

Reply:
Fixed here: 6bc9582db3f100429727234ecf27249cce9f5014

Reply:
I *think* only `Servo` should be capitalised. Indeed, I don't see "parallel browser engine project" at all on servo.org?

Reply:
Ah, yeah I took this from their github page. I'll remove it and just leave it as Servo

Reply:
Fixed: 2cbbbff5005e476a234e18147b6495e37d81c14f